conf/broker.conf

```diff
@@ -561,7 +561,8 @@ zookeeperSessionExpiredPolicy=shutdown
 # Enable or disable system topic
 systemTopicEnabled=false
 
-# The schema compatibility strategy to use for system topics
+# The schema compatibility strategy to use for system topics.
+# Available values: ALWAYS_INCOMPATIBLE, ALWAYS_COMPATIBLE, BACKWARD, FORWARD, FULL, BACKWARD_TRANSITIVE, FORWARD_TRANSITIVE, FULL_TRANSITIVE
```

Comment:
please keep consistent overall

```suggestion
# Available values: `ALWAYS_INCOMPATIBLE`, `ALWAYS_COMPATIBLE`, `BACKWARD`, `FORWARD`, `FULL`, `BACKWARD_TRANSITIVE`, `FORWARD_TRANSITIVE`, `FULL_TRANSITIVE`
```

Reply:
I think it's better to use `` in the document, it is highlight, but I don't want to use it in the conf file.